source/deployment-considerations/colocation.txt

```diff
@@ -0,0 +1,101 @@
+run Relational Migrator. Also note that the 
+:ref:`Kafka deployment model <kafka-intro>` allows you greater 
+resilience and scalability and can be used with any of the 
+deployment options.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use case 
+     - Recommended environment
+
+   * - Smaller scale jobs that run for less than 4 hours, general
+       debugging and testing.
+     - Labtop / Workstation / Docker
+
+   * - Larger scale production jobs that run for more than 4 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to run Relational Migrator
+--------------------------------
+
+For best performance, locate the machine or server running Relational 
+Migrator as geographically close to the source database as possible: 
+
+- If your using Relational Migrator for an on-premises migration, this 
+  means running Relational Migrator in the same data center as the source
+  database.
+
+- If your using a cloud hosted database this means running Relational 
+  Migrator on an EC2 instance or VM in the same 
+  :abbr:`VPC (Virtual Private Cloud)` as the source database.
+
+.. tip::
+
+   Check the sleep timeout setting for your operating system. If your 
+   machine goes to sleep in the middle of a migration the sync job fails. 
+
+Network Considerations
+----------------------
+
+When using Relational Migrator, the system firewall on the machine or 
+server must allow inbound and outbound TCP traffic to the source
+database. The source database connection must also allow TCP traffic 
+from the IPV4 address of the machine or server running Relational
+Migrator.
+
+VPC and Cloud Networking
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running Relational Migrator on an AWS EC2 instance, check the 
+security group and firewall configurations of the EC2 instance or VM. 
+Also check that the router table allows traffic to the database. If your 
+database is hosted on a public IP address, this means that the EC2 
+instance needs a router table entry to an internet gateway within the 
+:abbr:`VPC (Virtual Private Cloud)`. For details on specific cloud 
+provider network configurations see:
+
+- `AWS Networking <https://docs.aws.amazon.com/vpc/>`__
+- `Azure Networking <https://azure.microsoft.com/en-us/products/category/networking>`__
+- `GCP Networking <https://cloud.google.com/vpc/docs>`__
+
+AWS Private Link
+~~~~~~~~~~~~~~~~
+
+`AWS Private Link <https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html>`__ 
+is a robust, scalable solution allowing you to 
+establish private connections between your VPC and various services, 
+making them feel as though they're located within your own VPC. This 
+eliminates the necessity for an internet gateway or public IP addresses.
+
+You can use Relational Migrator with AWS private link. You need to 
+configure a
+`AWS private link endpoint <https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html>`__.
```

Comment:
Consider:

To use Relational Migrator with AWS Private Link, you need to configure...